lib/representer.ex

```diff
@@ -197,7 +197,8 @@ defmodule Representer do
     {{:., meta, path}, meta2, args}
   end
 
-  defp remove_type_parentheses({type, meta, args}) when args == [] or is_atom(args) do
+  defp remove_type_parentheses({type, meta, args})
+       when type != :%{} and (args == [] or is_atom(args)) do
```

Comment:
The issue was caused by us replacing `{%{}, [], []}` with `{%{}, [], nil}`, which apparently is not valid Elixir AST, or at least the formatter doesn't understand it.

Reply:
I suppose `{%{}, [], nil}` doesn't really make sense? 🤷 
Nice find